layout.tex

```diff
@@ -7,7 +7,7 @@
 
 \author{
 	Henrique Casares\\
-	Vini Lopes\\
+	Vinicius Lopes Silva\\
 	Lucas Campos Martins Ferreira Braga
 }
 \orientador{Prof. Dr. Marco Antônio Assis de Melo}
```

Comment:
Is it necessary change the name advisor ?

Reply:
We don't know yet all advisors, but we can use as _'placeholder'_

Reply:
I thought our advisor was Batatão